distribution/docs/src/main/resources/content/_coreConcepts/high-availability-supported-capabilities.adoc

```diff
@@ -0,0 +1,29 @@
+:type: subCoreConcept
+:status: published
+:title: High Availability Supported Capabilities
+:parent: Introduction to High Availability
+:order: 00
+
+===== {title}
+
+Only these capabilities are supported in a Highly Available Cluster. For an even more detailed view, see the <<_high_availability_supported_features,List of Supported Features>>.
+
+* User Interfaces:
+** Simple
+** Standard
+** Intrigue
+* Catalog:
+** Validation
+** Plug-ins: Expiration Date, JPEG2000, Metacard Validation, Schematron, Versioning
+** Transformers
+* Platform:
+** Actions
+** Configuration
+** Notifications
+** Persistence
+** Security: Audit, Encryption
+* Solr
+* Security
+* Endpoints:
+** REST Endpoint
+** CSW Endpoint
```

Comment:
❗️ We should add some of the third-party libraries (cxf, camel, etc.) since those are provided as part of the framework and are HA compatible.